cpp/src/io/orc/stripe_init.cu

```diff
@@ -127,14 +119,12 @@ extern "C" __global__ void __launch_bounds__(128, 8)
 {
   __shared__ compressed_stream_s strm_g[4];
 
-  volatile compressed_stream_s *const s = &strm_g[threadIdx.x >> 5];
-  int strm_id                           = blockIdx.x * 4 + (threadIdx.x >> 5);
-  int t                                 = threadIdx.x & 0x1f;
+  compressed_stream_s *const s = &strm_g[threadIdx.x >> 5];
+  int strm_id                  = blockIdx.x * 4 + (threadIdx.x >> 5);
+  int t                        = threadIdx.x & 0x1f;
```

Comment:
Same here. What's going on? Was this a latent bug

Reply:
When you meant bug, do you mean this https://github.com/rapidsai/cudf/pull/6559#discussion_r509831571 ?

Yes, it was.

Reply:
And I just removed unnecessary volatile usages in some places, which triggered rearrangement from clang.

Reply:
That's parquet. This is ORC. I think they're unrelated. And I think it wouldn't break now. Earlier the cooperative loaders were written thinking they would be read by the entire block but that might not be correct. Not since it's only one thread, instead of one thread of the entire block, this would be read by one thread of each warp.

Reply:
Ahh, I missed that. Will take care of it. I was in assumption that `t` was always thread ID, didn't observe it being used as lane id. Will check all other occurrences and correct them.